lib/logmsg/logmsg.c

```diff
@@ -524,9 +524,9 @@ log_msg_set_value(LogMessage *self, NVHandle handle, const gchar *value, gssize
   if (_log_name_value_updates(self))
     {
       msg_debug("Setting value",
-                evt_tag_printf("msg", "%p", self),
-                evt_tag_str("name", name),
-                evt_tag_printf("value", "%.*s", (gint) value_len, value));
+                evt_tag_str("macro", name),
+                evt_tag_printf("value", "%.*s", (gint) value_len, value),
+                evt_tag_printf("msg", "%p", self));
```

Comment:
The `evt_tag_printf(<name>, "%p", <pointer>)` construct is used in a fair amount of places. Wouldn't it make sense to introduce an `evt_tag_pointer`, that abstracted away the common parts?

An aesthetic change, but it would convey the intent better, it would be immediately clear what it does, without having to read the whole line.

Reply:
Agreed it would more readable.